genestack_client/file_types.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+
+class JavaFileTypes(object):
+
+    @classmethod
+    def is_file_type(cls, type_str):
+        types = {v for k, v in cls.__dict__.iteritems() if (not k.startswith("_") and isinstance(v, basestring))}
+        return type_str in types
+
+    _JAVA_PKG = "com.genestack.api.files."
+
+    AUXILIARY_FILE = _JAVA_PKG + "IAuxilaryFile"
+    FOLDER = _JAVA_PKG + "IFolder"
+    SEARCH_FOLDER = _JAVA_PKG + "ISearchFolder"
+    INDEX_FILE = _JAVA_PKG + "IIndexFile"
+
+    APPLICATION_PAGE_FILE = _JAVA_PKG + "IApplicationPageFile"
+    DICTIONARY_FILE = _JAVA_PKG + "IDictionaryFile"
+    PREFERENCES_FILE = _JAVA_PKG + "IPreferencesFile"
+    RAW_FILE = _JAVA_PKG + "IRawFile"
+    REPORT_FILE = _JAVA_PKG + "IReportFile"
+
+    _JAVA_BIO_PKG = "com.genestack.bio.files."
```

Comment:
Move technical constant to the top of file.

Space line separating does not add readability because you use them for two different proposes (split bio-core and split assays-other)

May be add commented lines:

```
# core file types

# bio file types
```